src/onesignal/OneSignal.ts

```diff
@@ -105,7 +105,7 @@ export default class OneSignal {
       throw new InvalidArgumentError('externalId', InvalidArgumentReason.WrongType);
     }
 
-    if (typeof jwtToken !== 'string') {
+    if (!!jwtToken && typeof jwtToken !== 'string') {
       throw new InvalidArgumentError('token', InvalidArgumentReason.WrongType);
```

Comment:
This is just a bit out of scope for the PR but I just noticed this error field name isn't correct. It should be `'jwtToken'` instead of `'token'` so it is more clear.